lib/base-compiler.ts

```diff
@@ -99,7 +99,8 @@ export class BaseCompiler implements ICompiler {
     protected toolchainPath: any;
     public possibleArguments: CompilerArguments;
     protected possibleTools: ITool[];
-    protected demanglerClass: any;
+    // TODO(jeremy-rifkin): Will be able to do better once #4607 lands
```

Comment:
#4607 has landed (but the TODO stands)

Reply:
Ah shoot I wasn't paying attention here, ty!